src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/HearingArrangements.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.hmcts.reform.sscs.domain.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.Objects;
+
+public class HearingArrangements {
+
+    private boolean languageInterpreter;
```

Comment:
Think this can be removed - we already have a HearingArrangements object in common library if needed